lib/pymedphys/_version.py

```diff
@@ -1,5 +1,5 @@
 # DO NOT EDIT THIS FILE!
-# This file has been autogenerated by `pymedphys dev propagate`
+# This file has been autogenerated by `poetry run pymedphys dev propagate`
 
 version_info = [0, 35, 0, "dev1"]
```

Comment:
does this need to be bumped to dev2 ?

Reply:
I haven't been strict on this. At the end of the day I have been bumping it just prior to a release.